src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/RSAPKCS1SignatureDescriptionBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Security.Cryptography.Xml
+{
+    internal abstract class RSAPKCS1SignatureDescriptionBase : SignatureDescription
```

Comment:
Why is this different from netfx? (Where it doesn't have the "Base" on the end).

https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/signaturedescription.cs,e4079280468587dc,references